src/fr/articles/html5-et-aria.md

```diff
@@ -8,42 +8,43 @@ tags:
+<abbr>ARIA</abbr> ne rajoute aucun comportement particulier ni aucune présentation visuelle aux composants d'interface. Il faut donc toujours assurer la mise en forme grâce aux <abbr>CSS</abbr>, le contrôle du contenu et la gestion de l'interactivité via JavaScript.
 
 ## Le support
 
-Le support d’<abbr>ARIA</abbr> s’améliore constamment et est déjà présent depuis longtemps&nbsp;:
+Le support d'<abbr>ARIA</abbr> s'améliore constamment et est déjà présent depuis longtemps&nbsp;:
 - Dans les navigateurs modernes (Firefox >=3.0, Chrome >=16, Safari >=4, Opera >=9.5, Internet Explorer 8 pour un support basique, Internet Explorer >=9 et Edge pour un support avancé).
-- Dans les aides techniques (<abbr>NVDA</abbr> >=2010, Orca >= 2 + Firefox uniquement, VoiceOver <abbr>OSX</abbr> >=10.5, <abbr>iOS</abbr> >=4, <abbr>JAWS</abbr> >=8, Window-Eyes >=7, ZoomText 10 pour un support basique, ZoomText >=11 pour un support avancé).  
+- Dans les aides techniques (<abbr>NVDA</abbr> >=2010, Orca >= 2 + Firefox uniquement, VoiceOver <abbr>OSX</abbr> >=10.5, <abbr>iOS</abbr> >=4, <abbr>JAWS</abbr> >=8, Window-Eyes >=7, ZoomText 10 pour un support basique, ZoomText >=11 pour un support avancé).
 
-De plus, l’efficacité optimale de la prise en charge d’<abbr>ARIA</abbr> dépend du couple navigateur&nbsp;/ aide technique&nbsp;: Firefox avec <abbr>NVDA</abbr>, Jaws avec Internet Explorer et Safari avec VoiceOver, par exemple.
+De plus, l'efficacité optimale de la prise en charge d'<abbr>ARIA</abbr> dépend du couple navigateur&nbsp;/ aide technique&nbsp;: Firefox avec <abbr>NVDA</abbr>, Jaws avec Internet Explorer et Safari avec VoiceOver, par exemple.
 
-## Faut-il l’utiliser&nbsp;?
+## Faut-il l'utiliser&nbsp;?
 
-Globalement, <abbr>ARIA</abbr> améliore l’accessibilité pour des composants pas nativement présents dans le <abbr>HTML</abbr> (ou pas accessibles notamment certains éléments <abbr>HTML5</abbr>). La robustesse et l’interopérabilité d’<abbr>ARIA</abbr> sont maintenant suffisantes pour qu’[une quarantaine de techniques WCAG 2.1 soient spécifiques à <abbr>ARIA</abbr>](https://www.w3.org/WAI/GL/wiki/Category:ARIA_Techniques) (contre 10 techniques <abbr>HTML5</abbr>). En conséquence, pour des applications Internet riches, l’<abbr>ARIA</abbr> est incontournable pour leur mise en accessibilité.
+Globalement, <abbr>ARIA</abbr> améliore l'accessibilité pour des composants qui ne sont pas nativement présents dans le <abbr>HTML</abbr> (ou pas accessibles notamment certains éléments <abbr>HTML5</abbr>). La robustesse et l'interopérabilité d'<abbr>ARIA</abbr> sont maintenant suffisantes pour qu'[une quarantaine de techniques WCAG 2.1 soient spécifiques à <abbr>ARIA</abbr>](https://www.w3.org/WAI/GL/wiki/Category:ARIA_Techniques) (contre 10 techniques <abbr>HTML5</abbr>). En conséquence, pour des applications Internet riches, l'<abbr>ARIA</abbr> est incontournable pour leur mise en accessibilité.
```

Comment:
Don't know exactly what to do whit this part of the text.
Techniques listed in https://www.w3.org/WAI/GL/wiki/Category:ARIA_Techniques and https://www.w3.org/WAI/WCAG21/Techniques/ are not the same.
Should we update it based on https://www.w3.org/WAI/WCAG22/Techniques/ ?